### PR TITLE
Replaced argument "album_type" with "include_groups" in "Spotify.artist_albums"

### LINKED ISSUE #1107: Get Artist's Albums now expects "include_groups" param instead of "album_type"
**Describe the bug**
Spotify Web API used to accept album_type param when getting artists albums, even though the docs say the name of the param is include_groups. Today they have updated the API to align with the docs, so the [album_type](https://github.com/spotipy-dev/spotipy/blob/028539e08a5d9d14646ff5118ab3bd80c7458c1f/spotipy/client.py#L423) param in artist_albums is no longer working. I have tested locally replacing the param with include_groups and it works. 

```
def artist_albums(self, artist_id, album_type=None, country=None, limit=20, offset=0):
        trid = self._get_id("artist", artist_id)
        return self._get(
            "artists/" + trid + "/albums",
            include_groups=album_type,
            country=country,
            limit=limit,
            offset=offset,
        )
```

This works. 

[Link to issue on Spotify dev community](https://community.spotify.com/t5/Spotify-for-Developers/Get-Artist-s-Albums-no-longer-accepts-quot-album-type-quot/m-p/6074975#M13945)

### PR DESCRIPTION
**Pull request by dieser-niko**
Fixes #1107

### CODE REVIEW

**Review by NickyReid — APPROVED** [review]
Thanks for the quick update, you're awesome

File: spotipy/client.py [hunk 1]
```diff
@@ -405,22 +405,33 @@ def artists(self, artists):
...
         """
 
+        if album_type:
+            warnings.warn(
+                "You're using `artist_albums(..., album_type='...')` which will be removed in "
+                "future versions. Please adjust your code accordingly by using "
+                "`artist_albums(..., include_groups='...')` instead.",
+                DeprecationWarning,
+            )
+            include_groups = album_type if include_groups is None else include_groups
```
Comment (stephanebruckert) [inline spotipy/client.py:429]:
```suggestion
            include_groups = include_groups or album_type
```

**Review by stephanebruckert — APPROVED** [review]
LGTM! Just added a small suggestion

**Comment by NickyReid** [comment]
@dieser-niko Has this fix been published to PyPi?

**Comment by dieser-niko** [comment]
Not yet. I have only recently become a member of spotipy and not everything has been clarified (documentation).

**Comment by NickyReid** [comment]
Ah okay, thanks 😄

**Pull request merged**